deploy/olm-catalog/knative-eventing-operator/0.11.0/knative-eventing-operator.v0.11.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,202 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: knative-eventing-operator.v0.11.0
+  namespace: placeholder
+  annotations:
+    alm-examples: >-
+      [{"apiVersion":"operator.knative.dev/v1alpha1","kind":"Eventing","metadata":{"name":"eventing"},"spec":{}},{"apiVersion":"operator.knative.dev/v1alpha1","kind":"Eventing","metadata":{"name":"eventing"},"spec":{}}]
+    categories: 'Networking,Integration & Delivery,Cloud Provider,Developer Tools'
+    certified: 'false'
+    description: |-
+      Knative Eventing is a system that is designed to address a
+      common need for cloud native development and provides composable
+      primitives to enable late-binding event sources and event
+      consumers.
+    containerImage: >-
+      gcr.io/knative-releases/knative.dev/eventing-operator/cmd/manager:v0.11.0
+    support: The Knative Authors
+    capabilities: Basic Install
+    repository: 'https://github.com/knative/eventing-operator'
+    createdAt: '2019-12-17T10:00:00Z'
+spec:
+  displayName: Knative Eventing Operator
+  description: >+
+    Knative Eventing is designed to address a common need for cloud
+    native development:
+
+    1. Services are loosely coupled during development and deployed
+    independently
+    1. A producer can generate events before a consumer is listening, and a
+    consumer
+       can express an interest in an event or class of events that is not yet being
+       produced.
+    1. Services can be connected to create new applications
+       - without modifying producer or consumer, and
+       - with the ability to select a specific subset of events from a particular
+         producer.
+
+    For complete Knative Eventing documentation, see
+    [Knative eventing](https://www.knative.dev/docs/eventing/) or
+    [Knative docs](https://www.knative.dev/docs/) to learn about Knative.
+  maturity: alpha
+  version: 0.11.0
+  minKubeVersion: ''
+  keywords:
+    - serverless
+    - eventing
+    - camel
```

Comment:
there is no camel, nor kafka

Reply:
+1, let me get rid of those here and also in operatorHub